tensorflow/core/grappler/utils/pattern_utils.cc

```diff
@@ -15,10 +15,22 @@ limitations under the License.
 
 #include "tensorflow/core/grappler/utils/pattern_utils.h"
 
+#include <algorithm>
+
 namespace tensorflow {
 namespace grappler {
 namespace utils {
 
+inline bool IsCommutativeOp(string op) {
+  // TODO(intel-tf): Add more ops to this list if needed.
+  static const std::vector<string> commutative_ops = {"Add", "AddV2", "Mul"};
```

Comment:
We do not allow statics with non trivial destructors in tf, also it could be an `absl::flath_hash_set`. The TF idiom for these things is `static const auto* commutative_ops = new absl::flat_hash_set(......)`

Reply:
@ezhulenev Thanks for the comment! I have addressed with your suggestion. Please check.